app/frontend/styles/application.scss

```diff
@@ -9,6 +9,8 @@
 $govuk-font-url-function: frontend-font-url;
 $govuk-image-url-function: frontend-image-url;
 
+$govuk-global-styles: true;
```

Comment:
What does setting this property do?

Reply:
It turns on global styles for P tags and A tags. This means you don't have to, for example add `govuk-body` and ‘govuk-link’ respectively onto every link to get the right styles (including focus styles).

It also makes it easy to copy styles from the prototype into production (which we sometimes do) which works the same way.

Reply:
I might be wary of turning that on. It does catch some things, but in general I've been trying to always use the right classes.

Reply:
I suggested turning this on once too… there aspects of the design system that really bug me, and while I totally understand the reasoning, having to add a class to every element to get standard styling is up there on my list of pet peeves!

Reply:
In fact, turning on these styles breaks the design of the tile component… link styles and focus state are broken.

<img width="486" alt="Screenshot 2021-02-22 at 10 34 52" src="https://user-images.githubusercontent.com/813383/108696584-a773e080-74f9-11eb-92d7-ac6ec61d213e.png">

Reply:
I'm wary of turning it on. The global styles mode has had some bugs in the past (as I don't think it’s routinely tested by the Design System team), and was apparently mainly done to make prototyping easier.

Maybe we can find other ways (like the rubocop rule) to catch elements which don't use the helpers that add the classes?

(I also dislike having to add a class to every element though. Especially as there are some which don't require it, like `<li>`...)

Reply:
I have removed.